plugins/woocommerce-admin/client/core-profiler/index.tsx

```diff
@@ -322,10 +327,16 @@ const handleGeolocation = assign( {
 } );
 
 const redirectToWooHome = () => {
-	/**
-	 * @todo replace with navigateTo
-	 */
-	window.location.href = '/wp-admin/admin.php?page=wc-admin';
+	navigateTo( {
+		url: getNewPath( {}, '/', {} ),
+	} );
+};
+
+const redirectToJetpackAuthPage = (
+	_context: CoreProfilerStateMachineContext,
+	event: { data: { url: string } }
+) => {
+	window.location.href = event.data.url;
```

Comment:
```suggestion
	window.location.href = event.data.url + '&installed_ext_success=1';
```
Or we can add this param in backend

Reply:
I think we can add it here.